src/config/contributors.json

```
@@ -0,0 +1,4295 @@
+        "translators"
+      ]
+    }
+  },
+  "lvandeve": {
+    "github": "lvandeve",
+    "name": "Lode Vandevenne",
+    "teams": {
+      "2021": [
+        "authors"
+      ]
+    }
+  },
+  "malchata": {
+    "avatar_url": "1635238",
+    "github": "malchata",
+    "name": "Jeremy Wagner",
+    "teams": {
+      "2022": [
+        "authors"
+      ]
+    },
+    "twitter": "malchata",
+    "website": "https://jlwagner.net/"
+  },
+  "mandymichael": {
+    "avatar_url": "11938879",
+    "github": "mandymichael",
+    "name": "Mandy Michael",
+    "teams": {
+      "2020": [
+        "reviewers"
+      ]
+    },
+    "twitter": "Mandy_Kerr",
+    "website": "https://mandymichael.com/"
+  },
+  "mathiasbynens": {
+    "avatar_url": "81942",
+    "github": "mathiasbynens",
+    "name": "Mathias Bynens",
+    "teams": {
+      "2019": [
+        "brainstormers",
+        "reviewers"
+      ]
+    },
+    "twitter": "mathias",
+    "website": "https://mathiasbynens.be/"
+  },
+  "matthewp": {
+    "avatar_url": "361671",
+    "github": "matthewp",
+    "name": "Matthew Phillips",
+    "teams": {
+      "2019": [
+        "reviewers"
+      ]
+    }
+  },
+  "mattludwig": {
+    "github": "mattludwig",
+    "name": "Matt Ludwig",
+    "teams": {
+      "2019": [
+        "reviewers"
+      ]
+    }
+  },
+  "matuzo": {
+    "avatar_url": "1496761",
+    "github": "matuzo",
+    "name": "Manuel Matuzovi\u0107",
+    "teams": {
+      "2020": [
+        "reviewers"
+      ]
+    },
+    "twitter": "mmatuzo",
+    "website": "https://www.matuzo.at/"
+  },
+  "maudnals": {
+    "avatar_url": "9762897",
+    "github": "maudnals",
+    "name": "Maud Nalpas",
+    "teams": {
+      "2021": [
+        "reviewers"
+      ]
+    }
+  },
+  "max-ostapenko": {
+    "avatar_url": "1611259",
+    "github": "max-ostapenko",
+    "name": "Max Ostapenko",
+    "teams": {
+      "2020": [
+        "analysts",
+        "developers"
+      ],
+      "2021": [
+        "analysts"
+      ],
+      "2022": [
+        "analysts"
+      ]
+    },
+    "twitter": "themax_o",
+    "website": "https://maxostapenko.com"
+  },
+  "mcmd": {
+    "avatar_url": "868584",
+    "github": "mcmd",
+    "linkedin": "miguelcarlosmartinezdiaz",
+    "name": "Miguel Carlos Mart\u00ednez D\u00edaz",
+    "teams": {
+      "2020": [
+        "translators"
+      ]
+    },
+    "twitter": "mcmd"
+  },
+  "mdiblasio": {
+    "avatar_url": "17748781",
+    "github": "mdiblasio",
+    "name": "Michael DiBlasio",
+    "teams": {
+      "2020": [
+        "authors"
+      ]
+    }
+  },
+  "mel-ada": {
+    "avatar_url": "5370850",
+    "github": "mel-ada",
+    "linkedin": "mel-ada",
+    "name": "Melissa Ada",
+    "teams": {
+      "2022": [
+        "authors"
+      ]
+    },
+    "twitter": "mel_melificent"
+  },
+  "meyerweb": {
+    "avatar_url": "939727",
+    "github": "meyerweb",
+    "name": "Eric A. Meyer",
+    "teams": {
+      "2019": [
+        "reviewers",
+        "editors"
```

This looks to be a mistake - see #3400

```suggestion
        "reviewers"
```